PYME/IO/image.py

```diff
@@ -1270,6 +1314,11 @@ def Load(self, filename=None, prompt=None, haveGUI = False):
                 self._loadNPY(filename)
             elif filename.endswith('.dbl'): #treat this as being an image series
                 self._loadDBL(filename)
+            elif '.obf' in os.path.splitext(filename)[1] or '.msr' in os.path.splitext(filename)[1]:
+                self._loadOBF(filename)
+                # hack to get around self.filename = filename below, otherwise if we open multiple
```

Comment:
Won't that work with the query string?

Reply:
Wasn't working for me. The query string gets parsed inside the load sub methods (e.g. loadOBF or the bioformats one), but at the bottom of `Load` proper, there's a `self.filename=filename`, so whatever went in originally is the stored one. Note that I'm not so sure what `self.seriesName` is really up to - I thought changing seriesName would do it, but it looks like we use filename for the window title / grabbing images for making composites.